indico/modules/events/papers/controllers/display.py

```diff
@@ -159,3 +162,20 @@ class RHDeletePaperComment(RHPaperCommentBase):
     def _process(self):
         delete_comment(self.comment)
         return jsonify_data(flash=False)
+
+
+class RHDisplayReviewingArea(RHPapersBase):
+    def _checkProtection(self):
+        if not session.user:
+            raise Forbidden
+        if session.user not in self.event_new.cfp.layout_reviewers and \
```

Comment:
we usually use () to wrap instead of `\` (the, the double indentation is correct):

```
if (session.user not in self.event_new.cfp.layout_reviewers and
        session.user not in self.event_new.cfp.content_reviewers):
    raise Forbidden
```